string.h header is missing in avahi-core/fdutil.c

37375541 FD_ZERO in sys/select.h should not rely on having memset() defined

--- a/avahi-core/fdutil.c
+++ b/avahi-core/fdutil.c	2024-11-27 11:01:07.561271046 +0100
@@ -25,6 +25,7 @@
 #include <unistd.h>
 #include <fcntl.h>
 #include <assert.h>
+#include <string.h>
 
 #include "fdutil.h"
 
